### PR TITLE
Support custom prefix

### PR DESCRIPTION
**Pull request by sahith-narahari**
Closes: #87

### CODE REVIEW

File: cosmos/launchpad/construction_derive_test.go [hunk 1]
```diff
@@ -50,3 +50,32 @@ func TestLaunchpad_ConstructionDerive(t *testing.T) {
 	})
 	require.Equal(t, ErrUnsupportedCurve, deriveErr)
 }
+
+func TestRandomeAddrPrefix(t *testing.T) {
```
Comment (alessio) [inline cosmos/launchpad/construction_derive_test.go:54]:
```suggestion
func TestRandomAddrPrefix(t *testing.T) {
```

File: cosmos/launchpad/construction_derive_test.go [hunk 2]
```diff
@@ -50,3 +50,32 @@ func TestLaunchpad_ConstructionDerive(t *testing.T) {
...
+	// Check using non cosmos address
+	adapter := NewLaunchpad(sdk.NewClient(""), tendermint.NewClient(""), properties)
+	deriveResp, deriveErr := adapter.ConstructionDerive(context.Background(), &types.ConstructionDeriveRequest{
+		PublicKey: &types.PublicKey{
+			Bytes:     data,
+			CurveType: "secp256k1",
+		},
+	})
+
+	require.Nil(t, deriveErr)
```
Comment (alessio) [inline cosmos/launchpad/construction_derive_test.go:77]:
```suggestion
	require.NoError(t, deriveErr)
```

Comment (sahith-narahari) [reply cosmos/launchpad/construction_derive_test.go:77]: rosetta errors don't satisy error interface

Comment (jgimeno) [reply cosmos/launchpad/construction_derive_test.go:77]: Yeah, that is correct, it cannot be done by noerror.

File: cosmos/launchpad/construction_derive.go [hunk 3]
```diff
@@ -4,15 +4,20 @@ import (
...
-	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 func (l Launchpad) ConstructionDerive(ctx context.Context, r *types.ConstructionDeriveRequest) (*types.ConstructionDeriveResponse, *types.Error) {
 	if r.PublicKey.CurveType != "secp256k1" {
 		return nil, ErrUnsupportedCurve
 	}
 
+	config := sdk.GetConfig()
```
Comment (jgimeno) [inline cosmos/launchpad/construction_derive.go:15]: I am not sure, but I don't think this is the correct point to do it. Given it is a singleton I think the SetBech32 prefixx should be done when the application is called. Maybe on main.

Comment (sahith-narahari) [reply cosmos/launchpad/construction_derive.go]: yeah moved it to launchpad constructor

**Review by jgimeno — APPROVED** [review]

**Review by alessio — APPROVED** [review]

**Pull request merged**